src/clusterpost-execution/engine_awsecs.js

```diff
@@ -0,0 +1,243 @@
+
+module.exports = function (conf) {
+
+	var fs = require('fs');
+	var Promise = require('bluebird');
+	var _ = require("underscore");
+	var spawn = require('child_process').spawn;
+	var path = require('path');
+	var Joi = require('@hapi/joi');
+	const { ECSClient, ListClustersCommand, RunTaskCommand, DescribeTasksCommand, StopTaskCommand } = require("@aws-sdk/client-ecs");
+
+
+	var executionmethods = require('./executionserver.methods')(conf);
+	var clustermodel = require('clusterpost-model');
+
+	var handler = {};
+
+	handler.submitJob = function(doc, cwd){
+
+		Joi.assert(doc, clustermodel.job);
+		var jobid = doc._id;
+		console.log("Job ID: " + jobid);
+		console.log("URI: " + conf.uri);
+		console.log("Token: " + conf.token);
+		return new Promise(function(resolve, reject){
+			// var command = 'aws';
+			// //aws ecs create-service --cluster fly-by-cnn --service-name fly-by-cnn-service --task-definition fly-by-docker:4 --desired-count 1 --launch-type "FARGATE" --network-configuration "awsvpcConfiguration={subnets=[subnet-0c8cf4101362bdd63],securityGroups=[sg-042c3b29e1aa0a65c],assignPublicIp=ENABLED}"
+			// var parameters = ['ecs', 'run-task', '--cluster', 'fly-by-cnn', '--task-definition',
+			// 		 'fly-by-docker:4', '--launch-type', 'FARGATE', '--network-configuration',
+			// 		  'awsvpcConfiguration={subnets=[subnet-0c8cf4101362bdd63],securityGroups=[sg-042c3b29e1aa0a65c],assignPublicIp=ENABLED}',
+			// 		  '--overrides', 'containerOverrides={name=fly-by-docker-con, command=[echo ' + jobid + ']}'
+			// 		];
+
+			// Set the AWS region
+			const REGION = "us-east-1";
+
+			// Create an S3 client service object
+			const ecs = new ECSClient({ region: REGION });
+
+			//--cluster fly-by-cnn --launch-type "FARGATE" --network-configuration "awsvpcConfiguration={subnets=[subnet-0c8cf4101362bdd63],securityGroups=[sg-042c3b29e1aa0a65c],assignPublicIp=ENABLED}" --task-definition fly-by-docker:4 --overrides "containerOverrides={name=fly-by-docker-con, command=[echo something]}"
+
+			const params = {
```

Comment:
Should this be hardcoded? 
This should be part of the software configuration. We could add all this description when we create new software.